File: hw/paillacho/U3/HW20Composite/Composite/src/main/java/ec/edu/espe/composite/model/Employee.java
package ec.edu.espe.composite.model;

/**
 *
 * @author Carlos Paillacho
 */
public abstract class Employee {
    protected String name = "not assigned yet";
    protected String title = "not assigned yet";

    /**
     * Prints the title and name of the employee.
     */
    public void stateName() {
        System.out.println(title + " " + name);
    }
}

File: hw/paillacho/U3/HW20Composite/CompositePy/model/Employee.py
"""
Base Employee class
@author Carlos Paillacho
"""


class Employee:
    def __init__(self, name: str = None):
        self.name = name or "not assigned yet"
        self.title = "not assigned yet"

    def state_name(self):
        # Prints title and name
        print(f"{self.title} {self.name}")
